### PR TITLE
[fix] scss - Bottom section on compact

### PR DESCRIPTION
**Pull request by sanlucca**
Bottom section on compact navSide has its style.
Fixing few bugs remaining on this menu to reduce shame file in our solutions

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://442-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://442-pr-lucca-front-LuccaSA.surge.sh

File: packages/scss/src/components/_nav-side.scss [hunk 1]
```diff
@@ -360,14 +371,22 @@
...
+		top: 0;
+		z-index: 1;
 	}
 
 	.navSide-bottomSection {
 		background: transparent;
 		position: relative;
 		width: auto;
 		height: auto;
+		&.is-elevated {
```
Comment (jeremie-lucca) [inline packages/scss/src/components/_nav-side.scss]: Should we remove this?

Comment (sanlucca) [reply packages/scss/src/components/_nav-side.scss]: Yeah

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://442-pr-lucca-front-LuccaSA.surge.sh

**Pull request merged**